src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,141 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import type { ReactElement } from 'react';
+import { cloneElement } from 'react';
+import { merge } from '@utilities/merge';
+import { IconSize } from '@foundation/Icon';
+import { HelperText } from '@utilities/input';
+import { Validation } from '@utilities/validation';
+import { generateRandomId } from '@utilities/generateRandomId';
+import { Tooltip, TooltipProps } from '@components/Tooltip/Tooltip';
+
+export type LabelProps = {
+    text: string;
+    hugWidth?: boolean;
+    required?: boolean;
+    secondaryLabel?: string;
+    tooltips?: TooltipProps[];
+    toolTipSize?: IconSize;
+};
+
+export type FormFieldProps = {
+    children: ReactElement;
+    label: LabelProps;
+    disabled?: boolean;
+    error?: boolean;
+    errorText?: string;
+    errorTextColor?: Validation;
+    helperText?: string;
+    helperTextColor?: Validation;
+    hiddenLabel?: boolean;
+    readOnly?: boolean;
+    status?: Validation;
+    'data-test-id'?: string;
+};
+
+export const FormField = ({
+    children,
+    disabled = false,
+    error = false,
+    errorText,
+    errorTextColor = Validation.Error,
+    helperText,
+    helperTextColor = Validation.Default,
+    hiddenLabel = false,
+    label,
+    readOnly = false,
+    status = Validation.Default,
+    'data-test-id': dataTestId = 'fondue-form-field',
+}: FormFieldProps): ReactElement => {
+    const getFormattedLabel = (): ReactElement => {
+        const { text, required, secondaryLabel, hugWidth, tooltips } = label;
+        const elements: ReactElement[] = [];
+        const secondaryLabelStyle = 'tw-w-full tw-flex tw-flex-nowrap tw-justify-between';
+        const applySecondaryLabel = !hugWidth && secondaryLabel;
+        let formattedLabel = text;
+
+        if (required) {
+            formattedLabel += ' *';
+        }
+        if (tooltips?.length) {
+            const tooltipElements: ReactElement[] = tooltips.map((tip: TooltipProps) => (
+                <Tooltip
+                    key={`form-field-tooltip-${generateRandomId()}`}
+                    content={tip.content}
+                    size={tip.size}
+                    data-test-id={tip['data-test-id']}
+                >
+                    {tip.children}
+                </Tooltip>
+            ));
+            elements.push(
+                <span className={'tw-flex tw-gap-2'}>
+                    {formattedLabel}
+                    {...tooltipElements}
```

Comment:
```suggestion
                   <FormatTooltip />
```
And then here call it.